src/css/wizard-navigation-bar.scss

```diff
@@ -140,120 +120,155 @@ $wz-param-indicator-color: null !global;
       }
 
       $wz-param-indicator-state: 'default' !global;
```

Comment:
I may have asked this before, but what exactly does `!global` do?

Reply:
I think you missed this comment

Reply:
Yes, missed this comment, my bad :( `!global` tells SCSS that we are referencing the global variable `$wz-param-indicator-state`. Without this flag, SCSS would introduce a new local variable with the same name, which would not be visible in the mixin included below.